bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -67,6 +67,8 @@ public class Bootstrap
 
     private Map<String, String> requiredConfigurationProperties;
     private Map<String, String> optionalConfigurationProperties;
+    private List<String> securitySensitivePatterns;
```

Comment:
Is pattern a regex? Below it looks like you are using `contains`. Can we have a good default list? So one does not need to configure whenever `Boostrap` is used?

Reply:
@kokosing it's as substring, not a pattern, I could make it a regex